internal/ethapi/api.go

```diff
@@ -1135,6 +1135,34 @@ func (api *BlockChainAPI) CreateAccessList(ctx context.Context, args Transaction
 	return result, nil
 }
 
+type config struct {
+	ActivationTime  uint64                    `json:"activationTime"`
+	BlobSchedule    *params.BlobConfig        `json:"blobSchedule"`
```

Comment:
```suggestion
	BlobSchedule    *params.BlobConfig        `json:"blobsSchedule"`
```
According to the spec this should be blobsSchedule, I think we should change the spec here though...

Reply:
Looks like naming was fixed in the spec!